### PR TITLE
[DEV-1599] EVM swaps

### PR DESCRIPTION
**Pull request by davidleomay**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations
  - Login/logout
  - Registration
  - Update user data
  - Create/delete payment routes

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

**Review by Yannick1712 — APPROVED** [review]

File: infrastructure/bicep/dfx-api.bicep
Comment (davidleomay) [inline infrastructure/bicep/dfx-api.bicep:1]: Base + Polygon?

File: src/shared/models/asset/asset.entity.ts
Comment (davidleomay) [inline src/shared/models/asset/asset.entity.ts:1]: Use approxPriceChf, check usages on develop?

File: src/subdomains/core/buy-crypto/process/entities/buy-crypto.entity.ts
Comment (davidleomay) [inline src/subdomains/core/buy-crypto/process/entities/buy-crypto.entity.ts:1]: Revert

File: src/subdomains/supporting/dex/dex.module.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/dex.module.ts:1]: Revert

File: src/subdomains/supporting/dex/services/dex-defichain.service.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/services/dex-defichain.service.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/check-liquidity/impl/defichain-default.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/check-liquidity/impl/defichain-default.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/check-liquidity/impl/defichain-poolpair.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/check-liquidity/impl/defichain-poolpair.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/check-liquidity/impl/defichain.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/check-liquidity/impl/defichain.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/base/defichain-non-poolpair.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/base/defichain-non-poolpair.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/defichain-crypto.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/defichain-crypto.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/defichain-dfi.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/defichain-dfi.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/defichain-poolpair.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/defichain-poolpair.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/defichain-stock.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/purchase-liquidity/impl/defichain-stock.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/sell-liquidity/impl/base/defichain.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/sell-liquidity/impl/base/defichain.strategy.ts:1]: Revert

File: src/subdomains/supporting/dex/strategies/sell-liquidity/impl/defichain-token.strategy.ts
Comment (davidleomay) [inline src/subdomains/supporting/dex/strategies/sell-liquidity/impl/defichain-token.strategy.ts:1]: Revert

File: src/subdomains/supporting/pricing/services/integration/pricing-defichain.service.ts
Comment (davidleomay) [inline src/subdomains/supporting/pricing/services/integration/pricing-defichain.service.ts:1]: Revert

**Pull request closed without merging**